src/libstore/build/derivation-goal.cc

```diff
@@ -1175,10 +1175,10 @@ bool DerivationGoal::isReadDesc(int fd)
     return fd == hook->builderOut.readSide.get();
 }
 
-
 void DerivationGoal::handleChildOutput(int fd, const string & data)
 {
-    if (isReadDesc(fd))
+    auto isWrittenToLog = isReadDesc(fd);
+    if (isWrittenToLog)
```

Comment:
Is this for just the `ssh://` case, or also local building? Can you add a code comment with the answer?